src/docs/api/greencheck/01_v3/check-single-domain.md

```diff
@@ -0,0 +1,144 @@
+---
+tags: apiMain
+libraryName: Greencheck API
+title: v3 - Check a domain
+description: Use this API to check if a domain is hosted on a green web host recognised by The Green Web Foundation.
+hasTabs: true
+eleventyNavigation:
+  key: v3-check-single-domain
+  title: Check a domain
+  order: 1
+  # sectionTitle: Version 3
+---
+
+# Query for a single domain
+
+The Greencheck API provides endpoints that allow developers to query [The Green Web Foundation's dataset](https://datasets.thegreenwebfoundation.org/) of green domains.
+
+The endpoint allows you to check if a single website domain is served from a green web host.
+
+## Request method
+
+Use the `GET` method for this request.
+
+## Endpoint
+
+The request should be sent to `http://api.thegreenwebfoundation.org/api/v3/greencheck/[hostname]`.
```

Comment:
Having worked with OpenAPI specification before, I found the `[value]` notation a bit misleading. Normally in this context the API documentations I've came accross used `{value}` notation. I know that our API and docs don't necessarily follow any specification, I wanted to raise this point as something that I noticed